package.json

```diff
@@ -57,6 +56,7 @@
     "angular.validators": "~4.4.3",
     "array-includes": "~3.0.3",
     "awesome-debounce-promise": "^2.1.0",
+    "bootstrap": "^4.1.1",
```

Comment:
bootstrap 4 is not compatible with bootstrap 3, we're using bootstrap 3

Reply:
Yeah i will remove this, i don't think we need this, ui is working without adding Bootstrap, may be it is getting  added in other repos, by adding, i was just checking if testcase failures are related to this, thank you.